pkg/virt-api/webhooks/validating-webhook/admitters/vmrestore-admitter.go

```diff
@@ -179,27 +181,20 @@ func (admitter *VMRestoreAdmitter) Admit(ar *admissionv1.AdmissionReview) *admis
 	return &reviewResponse
 }
 
-func (admitter *VMRestoreAdmitter) validateCreateVM(field *k8sfield.Path, namespace, name string) ([]metav1.StatusCause, *types.UID, error) {
+func (admitter *VMRestoreAdmitter) validateCreateVM(field *k8sfield.Path, namespace, name string) (causes []metav1.StatusCause, uid *types.UID, targetVMExists bool, err error) {
 	vm, err := admitter.Client.VirtualMachine(namespace).Get(name, &metav1.GetOptions{})
 	if errors.IsNotFound(err) {
-		return []metav1.StatusCause{
-			{
-				Type:    metav1.CauseTypeFieldValueInvalid,
-				Message: fmt.Sprintf("VirtualMachine %q does not exist", name),
-				Field:   field.String(),
-			},
-		}, nil, nil
+		// If the target VM does not exist it would be automatically created by the restore controller
+		return nil, nil, false, nil
 	}
 
 	if err != nil {
-		return nil, nil, err
+		return nil, nil, true, err
```

Comment:
not sure here it should be true, even though it doesnt really effect anything

Reply:
You're right. Thanks! Will change that.